pages/osx/nm.md

```diff
@@ -13,3 +13,7 @@
 - List all symbols, even debugging symbols:
 
 `nm -a {{file.o}}`
+
+- Demangle C++ symbols:
```

Comment:
Can we do without the `-p` and `-i` ? If they are required / recommended, please explain what they do in the description.

Reply:
They can be removed and I've done that in a new commit.